yal/assessments.py

```diff
@@ -207,6 +208,14 @@ async def state_survey_process_answer(self):
             self.save_metadata("assessment_section", section + 1)
             self.save_metadata("assessment_question_nr", 1)
             self.delete_metadata("assessment_question")
+            # clear assessment reminder info
+            data = {
```

Comment:
Is it possible that these will be empty when a user gets here? Maybe there should be a check (eg against what is in the metadata) and you only do the update if they aren't empty